spring/pom.xml

```diff
@@ -0,0 +1,169 @@
+~ distribution for a full listing of individual contributors.
+~
+~ This is free software; you can redistribute it and/or modify it
+~ under the terms of the GNU Lesser General Public License as
+~ published by the Free Software Foundation; either version 2.1 of
+~ the License, or (at your option) any later version.
+~
+~ This software is distributed in the hope that it will be useful,
+~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+~ Lesser General Public License for more details.
+~
+~ You should have received a copy of the GNU Lesser General Public
+~ License along with this software; if not, write to the Free
+~ Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+~ 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.jboss.narayana.quickstarts.spring</groupId>
+    <artifactId>spring-all</artifactId>
+    <version>5.3.0.Final-SNAPSHOT</version>
+    <packaging>pom</packaging>
+
+    <name>Narayana Quickstarts: JBTM and Spring</name>
+    <description>Examples show how to integrate with Spring</description>
+
+    <url>http://jboss.org/jbosstm</url>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.compiler.target>1.7</maven.compiler.target>
+        <maven.compiler.source>1.7</maven.compiler.source>
+        <version.maven-compiler-plugin>3.1</version.maven-compiler-plugin>
+        <version.maven-surefire-plugin>2.9</version.maven-surefire-plugin>
+        <version.org.springframwork>4.2.0.RELEASE</version.org.springframwork>
+        <version.org.jboss.narayana>${project.version}</version.org.jboss.narayana>
+        <version.org.jboss.logging.jboss-logging>3.1.2.GA</version.org.jboss.logging.jboss-logging>
+        <version.log4j>1.2.16</version.log4j>
+        <version.org.jboss.spec.javax.transaction>1.0.1.Final</version.org.jboss.spec.javax.transaction>
+        <version.org.jboss.jboss-transaction-spi>7.1.1.Final</version.org.jboss.jboss-transaction-spi>
+        <version.com.h2database>1.3.171</version.com.h2database>
+        <version.junit>4.10</version.junit>
+    </properties>
+
+    <repositories>
+        <repository>
+            <id>jboss-public-repository-group</id>
+            <name>JBoss Public Maven Repository Group</name>
+            <url>https://repository.jboss.org/nexus/content/groups/public/</url>
+            <releases>
+                <enabled>true</enabled>
+            </releases>
+            <snapshots>
+                <enabled>true</enabled>
+            </snapshots>
+        </repository>
+    </repositories>
+
+    <dependencies>
```

Comment:
I though that we've agreed on making the quickstart self contained and moving dependencies with other required stuff to spring/jta/pom.xml? Because another spring quickstart might not need all these dependencies and properties.